### PR TITLE
[10.0] [mig] procurement_purchase_no_grouping: Migration to v10.0

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa

### CODE REVIEW

File: procurement_purchase_no_grouping/models/purchase_order.py [hunk 1]
```diff
@@ -3,37 +3,37 @@
...
-               context=None, count=False):
+    @api.model
+    def search(self, args, offset=0, limit=None, order=None, count=False):
         make_po_conditions = {
             'partner_id', 'state', 'picking_type_id', 'company_id',
             'dest_address_id',
         }
         # Restrict the empty return for these conditions
-        if (context and context.get('grouping', 'standard') == 'order' and
+        if (self._context and
```
Comment (pedrobaeza) [inline procurement_purchase_no_grouping/models/purchase_order.py]: Always use `self.env.context`

File: procurement_purchase_no_grouping/models/purchase_order.py [hunk 2]
```diff
@@ -3,37 +3,37 @@
...
 class PurchaseOrderLine(models.Model):
     _inherit = 'purchase.order.line'
 
-    def search(self, cr, uid, args, offset=0, limit=None, order=None,
-               context=None, count=False):
+    @api.model
+    def search(self, args, offset=0, limit=None, order=None, count=False):
         # Restrict the empty return for these conditions
-        if (context and context.get('grouping', 'standard') == 'line' and
+        if (self._context and
```
Comment (pedrobaeza) [inline procurement_purchase_no_grouping/models/purchase_order.py]: `self.env.context`

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by pedrobaeza** [comment]
@carlosdauden please review

**Review by carlosdauden — APPROVED** [review]
LGTM after replace _context by self.env.context
Test OK

**Comment by sergio-teruel** [comment]
@pedrobaeza @carlosdauden Changes done.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**